src/js/components/Form/stories/UnsolicitedFeedback.js

```diff
@@ -0,0 +1,133 @@
+import React, { useContext, useState } from 'react';
+import styled from 'styled-components';
+import {
+  Box,
+  Button,
+  Form,
+  Layer,
+  FormField,
+  Text,
+  TextArea,
+  Header,
+  Heading,
+  ThumbsRating,
+  ResponsiveContext,
+} from 'grommet';
+
+// create a floating button for story example
+const PositionedFeedbackButton = styled(Button)`
+  position: fixed;
+  bottom: 0px;
+  border-radius: 6px;
+  right: 0px;
+  z-index: 10;
+`;
+
+// This example shows a way to perform validation across multiple fields.
+export const UnSolicitedFeedback = () => {
+  const size = useContext(ResponsiveContext);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  const closeFeedbackModal = () => {
+    setTimeout(() => {
+      setOpen(false);
+      setIsSuccessful(false);
+    }, 3000);
+  };
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <>
+      <PositionedFeedbackButton
+        onClick={onOpen}
+        margin={{ vertical: 'medium', horizontal: 'medium' }}
+        elevation="large"
+        color="brand"
```

Comment:
Not sure why but in storybook with the hpe theme this button is showing up with dark text. Then when the Layer is open the button has light text. Seems like it should always have light text so there is enough contrast

<img width="180" alt="Screen Shot 2022-12-15 at 2 43 18 PM" src="https://user-images.githubusercontent.com/54560994/207973595-6fb57c78-b772-415b-87ce-734b1ae28f12.png">

<img width="180" alt="Screen Shot 2022-12-15 at 2 43 59 PM" src="https://user-images.githubusercontent.com/54560994/207973739-e4780a10-f3a1-4181-a8fe-7f232e97026c.png">

Reply:
yeah here is the issue for this 
https://github.com/grommet/grommet/issues/6301